### PR TITLE
I've reorginized the files on the master branch

### PR DESCRIPTION
**Pull request by dcuffari**
I like the way organized these files. I accept these changes.

### CODE REVIEW

**Review by dcuffari — COMMENTED** [review]
Good work! You don't suck after all.

**Review by kelle — CHANGES_REQUESTED** [review]
There are no tests implemented so this PR is not accepted.

**Comment by dcuffari** [comment]
@ECGonzales I've made the changes that Kelle requested. Please review these changes.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+- "2.7"
+
+script:
+- py.test
```
Comment (ECGonzales) [inline .travis.yml]: do you like getting all the travis emails when you FAIL? If not add
notifications:
email:false

File: simulation/gravsim.py [hunk 2]
```diff
@@ -21,13 +21,48 @@
...
 import sys, getopt	#modules used to fancify user interface.
 
 #####################################################################						    
 #run time argument handeling of user defined values.		    #
 #						    		    #
 #####################################################################
+
+
+def input_check(n):
+    """
```
Comment (ECGonzales) [inline simulation/gravsim.py]: Where's the docstring??

File: simulation/gravsim.py [hunk 3]
```diff
@@ -21,13 +21,48 @@
...
+        dt = sys.argv[6]
+        return filename, N, D, S, G, dt
+    else:
+        print '\nThe proper use of gravsim.py is as follows.'
+        print '\npython gravsim.py outputfile N D S G dt\n'
+        return sys.exit(2)
+    
+
+def input_int_check(N, D, S):
+    """
```
Comment (ECGonzales) [inline simulation/gravsim.py]: DOCSTRING!

File: .travis.yml (hunk 1 above)
Comment (dcuffari) [reply .travis.yml]: Thanks, I've made this change.

File: tests/test_book.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from book.book import *
+import pytest
+
+
+def test_book()
```
Comment (ECGonzales) [inline tests/test_book.py]: Is this finished? def needs colons after parenthesis.

File: simulation/gravsim.py (hunk 2 above)
Comment (dcuffari) [reply simulation/gravsim.py]: I've included the docstring.

File: simulation/gravsim.py (hunk 3 above)
Comment (dcuffari) [reply simulation/gravsim.py]: I've included the docstring.

File: tests/test_book.py (hunk 4 above)
Comment (dcuffari) [reply tests/test_book.py]: I've completed the test.

**Comment by ECGonzales** [comment]
This looks good to go now.

**Comment by kelle** [comment]
- do you expect the tests to be passing?
- it looks like you are tracking too many files with github. There should be no .pyc files nor anything in the .cache directory tracked by git.

**Review by kelle — CHANGES_REQUESTED** [review]
it looks better than before I see files in __pycache__ which shouldnt be tracked.

**Review by kelle — CHANGES_REQUESTED** [review]
remove book.pyc

**Comment by dcuffari** [comment]
I removed book.pyc please check again.

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+numpy.py
```
Comment (kelle) [inline requirements.txt]: should just be numpy.

**Comment by dcuffari** [comment]
I'm withdrawing my pull request. I need to make some branch changes and then I will re-issue a pull request. 

-Dave

**Pull request closed without merging**